src/reference/informative/informA1.rst

```diff
@@ -16,35 +16,39 @@
     In CellML, two namespaces are used.
     These are the CellML namespace itself (which helps to define the :code:`units` elements as distinct from the XML default units system) and the MathML namespace used to interpret the :code:`math` elements.
     For more information on namespaces in general, please refer to the :namespace_help:`W3 schools namespace page <>`, or for the MathML namespace please refer to the :mathml2help:`W3 MathML namespace <#inferf.namespace>` page.
-    
+
     .. container:: heading3
 
-      Unicode, Basic Latin, and European numerals
+      Unicode, Basic Latin alphabetical characters, and digits
 
     The Unicode project is an attempt to codify all of the symbols - alphabets, writings, even emojis - of all the languages of the world so that they can be interchangeable and interpreted by computers.
     Since computers understand numbers rather than symbols, each character in each language is given a unique numerical code.
-    The codes themselves are arranged into blocks representing sets or alphabets of characters, and the Basic Latin alphabet is one of these.
-    It contains the upper and lowercase alphabet without decoration:
+    The codes themselves are arranged into blocks representing sets of characters, and the Basic Latin block is one of these.
+    It contains, amongst other things, the upper and lowercase alphabet without decoration:
 
       :code:`abcdefghijklmnopqrstuvwxyz` (symbols between :unicode:`0061` and :unicode:`007A`)
       :code:`ABCDEFGHIJKLMNOPQRSTUVWXYZ` (symbols between :unicode:`0041` and :unicode:`005A`)
 
-    The European numerals are the Unicode set:
+    These characters are refferd to in CellML as the Basic Latin alphabetic characters.
+
+    The digits are the Unicode characters:
 
       :code:`0123456789` (symbols between :unicode:`0030` and :unicode:`0039`)
 
+    Together with the Basic Latin alphabetic characters, they form the Basic Latin alphanumerical characters.
+
     In addition, CellML recognises four special characters:
 
     - :code:`+` the plus sign (:unicode:`002B`),
     - :code:`-` the minus sign (:unicode:`002D`),
+    - :code:`.` the full stop or decimal point (:unicode:`002E`),
     - :code:`_` the underscore (:unicode:`005F`),
-    - :code:`.` the fullstop (:unicode:`002E`),
 
     and the following whitespace characters:
-    
-    - the space (:unicode:`0020`), 
-    - the tab (:unicode:`0009`), 
+
+    - the space (:unicode:`0020`),
+    - the tab (:unicode:`0009`),
     - the carriage return (:unicode:`000D`), and
-    - the line feed (:unicode:`000A`).  
+    - the line feed (:unicode:`000A`).
 
-    Together these symbols define the only characters which can be interpreted into values for CellML element attributes and content. 
+    Together these symbols define the only characters which can be interpreted into values for CellML names and numbers.
```

Comment:
It's not quite the same thing?  We can only use these characters anywhere that something needs to be interpreted, including references etc?  Hmmm ... and now thinking about it, what about all the possible symbols in URLs?? We haven't listed them anywhere ...?

Reply:
We refer to the xlink spec for URLs

> The value of this attribute SHALL be a valid locator href, as defined in Section 5.4 of the XLink specification.
> The href attribute SHALL be treated according to the XLink specification, by applying the rules for simple-type elements.

Reply:
Have replaced this sentence by something less restrictive. (Just saying these classes will be uesd, not that they will be used exclusively!)